index.php

```diff
@@ -1,3 +1,12 @@
+<?php
+
+session_start();
+
+require_once 'src/Services/NavBarService.php';
+echo NavBarService::displayNavBar();
```

Comment:
You should move this echo down so it's inside the body tag. Technically this is invalid HTML, all visible content should be within <body></body>